src/main/java/io/vertx/codegen/type/TypeInfo.java

```diff
@@ -102,6 +100,20 @@ public boolean isVariable() {
     return false;
   }
 
+  /**
+   * @return {@code true} if this type holds a {@link DataObjectInfo}
+   */
+  public boolean isDataObjectHolder() {
```

Comment:
that's what I did initially, but this collides with the `getDataObject()` method when it is used in the ruby MVEL template because `.dataObject` is ambiguous and means either `getDataObject()` or `isDataObject()`

Reply:
Maybe `hasDataObjectInfo`?

Reply:
calling it isDataObjectHolder() makes this a property that's why I used it - it makes sense to me, more than "has"